ecmascript/transforms/base/src/fixer.rs

```diff
@@ -389,6 +396,13 @@ impl VisitMut for Fixer<'_> {
         }
     }
 
+    fn visit_mut_stmt(&mut self, s: &mut Stmt) {
```

Comment:
This fixes a broken test once I added `Context::Callee` to the list of contexts to wrap with parens around a sequence expression. I think a previous statement was setting the context and it wasn't being reset when moving on to the next statement.